src/HTTPUpdateServer.h

```diff
@@ -29,26 +33,43 @@ static String serverIndex =
      </html>)";
 static const char* successResponse = "<META http-equiv=\"refresh\" content=\"10;URL=/\">Update Success! Rebooting...";
 static const char* cancelResponse = "<META http-equiv=\"refresh\" content=\"10;URL=/\">Rebooting...";
+#else
+// embedded new Website
+extern const uint8_t update_html_start[] asm("_binary_assets_minified_update_html_gz_start");
+extern const uint8_t update_html_end[] asm("_binary_assets_minified_update_html_gz_end");
+static const char* successResponse = "Update Success! Rebooting...";
+static const char* cancelResponse = "Rebooting...";
+#endif
 
 class HTTPUpdateServer {
   public:
-    static void setVersion(const char* version) {
```

Comment:
I prefer having these static functions  and source the version and chip id from main.cpp where they are already.